src/Search/Console.php

```diff
@@ -120,6 +120,10 @@ public function __construct(Application $console, Kernel $app)
         $this->app = $app;
         $this->root = __DIR__.'/../..';
 
+        if (!defined('GEARMAN_INSTALLED')) {
```

Comment:
I agree with keeping xdebug out, and also the Gearman daemon, but the extension is a set of classes, it shouldn't create a performance problem?